galsim/table.py

```diff
@@ -195,20 +195,20 @@ def __call__(self, x):
             if dimen > 2:
                 raise ValueError("Arrays with dimension larger than 2 not allowed!")
             elif dimen == 2:
-                f = np.empty_like(x.ravel()).astype(float)
-                self.table.interpMany(x.astype(float).ravel(),f)
+                f = np.empty_like(x.ravel(), dtype=float)
```

Comment:
*And by "a code snippet" I mean code that shows where I got the 60% statement from.